app/web/.env.development

```diff
@@ -4,6 +4,4 @@ NEXT_PUBLIC_MEDIA_BASE_URL="https://dev-user-media.couchershq.org"
 NEXT_PUBLIC_CONSOLE_BASE_URL="https://next-console.couchershq.org"
 NEXT_PUBLIC_MAPBOX_KEY="pk.eyJ1IjoiY291Y2hlcnMiLCJhIjoiY2tpYzY4eHd2MGVpcTJ0bGdhdGhvbHhlbiJ9.u5zvDeFE9H8itTK_dNp7Pg"
 NEXT_PUBLIC_NOMINATIM_URL="https://nominatim.openstreetmap.org/"
-NEXT_PUBLIC_TILE_URL="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
-NEXT_PUBLIC_TILE_ATTRIBUTION="Map data &copy; <a href="https://osm.org/copyright">OpenStreetMap</a> contributors"
-NEXT_PUBLIC_STRIPE_KEY="pk_test_51KEzByIfR5z29g5khFE5samD8XKOGLcCrM1lhCkfOomGPUFAEYOw8uAqI2Nkv33wYdPM2FgTQNTC07IiNfHY1kLJ00Jqm8Ppai"
\ No newline at end of file
+NEXT_PUBLIC_STRIPE_KEY="pk_test_51KEzByIfR5z29g5khFE5samD8XKOGLcCrM1lhCkfOomGPUFAEYOw8uAqI2Nkv33wYdPM2FgTQNTC07IiNfHY1kLJ00Jqm8Ppai"
```

Comment:
Does this have to do with the app? If other cleanup might be better to break into another PR. Then if anything goes wrong easier to revert it.

Reply:
Moved to #4690